### ISSUE TITLE
Is there delay between last curve point and click action?

### ISSUE THREAD

**Issue by AndriusCTR** [Xetera/ghost-cursor#112]
Great plugin, but since the docs are very minimal, I wonder if there is a setting (or better yet, random time delay implementation) so that when cursor moves over a button, it waits 'a bit' (like human) before clicking it.

**Comment by TheGP** [comment]
Seems like there is not:
```      if (selector !== undefined) {
        await actions.move(selector, options)
        actions.toggleRandomMove(false)
      }

      try {
        await page.mouse.down()```

**Comment by Niek** [comment]
Fixed in #116

**Comment by AndriusCTR** [comment]
> Fixed in #116

Is this a random delay? If it's fixed then it's the same as no delay... definitely need to have random number

**Comment by AndriusCTR** [comment]
> @AndriusCTR Generate the random number and pass it. This is a cursor library, not a random numbers generator library.

Thanks, somehow missed that it's an options setting. Is this official release?
